### PR TITLE
RATIS-1349. Update the website of ratis 2.0.0

### PR DESCRIPTION
**Pull request by runzhiwang**

### CODE REVIEW

**Comment by runzhiwang** [comment]
@szetszwo Could you help review this ? Thanks a lot.

File: content/post/2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+[Download](https://ratis.apache.org/downloads.html)
+
+It contains around 260 improvements and bug fixes based on various Apache Hadoop Ozone use cases.
+See the [changes between 1.0.0 and 2.0.0](https://github.com/apache/ratis/compare/ratis-1.0.0...ratis-2.0.0-rc4) releases.
```
Comment (szetszwo) [inline content/post/2.0.0.md:24]: Please create a ratis-2.0.0 tag.  We should not use the rc4 tag for the official release.

**Review by szetszwo — COMMENTED** [review]
@runzhiwang , thanks a lot.  A minor comment git tag.

**Comment by szetszwo** [comment]
Btw, we should use "Apache Ozone" instead of "Apache Hadoop Ozone" in 2.0.0.md line 23.

**Comment by runzhiwang** [comment]
@szetszwo Thanks for review. I have updated the patch.

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Pull request merged**